internal/oci/container_test.go

```diff
@@ -238,4 +238,38 @@ var _ = t.Describe("Container", func() {
 		// Then
 		Expect(err).To(BeNil())
 	})
+	It("should be false if pid unintialized", func() {
+		// Given
+		state := &oci.ContainerState{}
+		state.Pid = 0
+		sut.SetState(state)
+		// When
+		err := sut.IsRunning()
+
+		// Then
+		Expect(err).To(Equal(false))
+	})
+	It("should succeed if pid is running", func() {
+		// Given
+		state := &oci.ContainerState{}
+		state.Pid = 1
```

Comment:
If this mean to `kill(1, 0)` (ie check if pid 1 (init) is running), this will fail with EPERM if the caller is not root -- so the test case should require root.

Reply:
personally, I've had trouble running these tests as nonroot anyway

Reply:
You mean, the already existing test cases are failing when run as non-root, so there's no point caring about the ones you add?

Fine my be then.